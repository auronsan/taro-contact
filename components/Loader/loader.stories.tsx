import React from 'react';
import { Loader } from './index';

export default {
  title: 'components/Loader',
  component: Loader,
};

export const Default = () => <Loader />;

export const WithProps = () => <Loader className="custom-class" />;
