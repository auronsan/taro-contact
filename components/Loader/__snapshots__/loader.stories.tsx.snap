// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/Loader Default smoke-test 1`] = `
<main class="roboto-normal">
  <div style="display: flex; justify-content: center; align-items: center; height: 100vh; flex-direction: column;">
    <span class="loader_root__PzLDe loader_barsLoader__m37dO">
      <span class="loader_bar__7sZw3">
      </span>
      <span class="loader_bar__7sZw3">
      </span>
      <span class="loader_bar__7sZw3">
      </span>
    </span>
  </div>
  <div class="stack_root__KVCbK toast_toast-container__8fZtQ stack_gap-xs__xTnny"
       id="toast-container"
       data-testid="toast-container"
       style="padding: 0px;"
  >
  </div>
</main>
`;

exports[`components/Loader WithProps smoke-test 1`] = `
<main class="roboto-normal">
  <div style="display: flex; justify-content: center; align-items: center; height: 100vh; flex-direction: column;">
    <span class="loader_root__PzLDe loader_barsLoader__m37dO custom-class">
      <span class="loader_bar__7sZw3">
      </span>
      <span class="loader_bar__7sZw3">
      </span>
      <span class="loader_bar__7sZw3">
      </span>
    </span>
  </div>
  <div class="stack_root__KVCbK toast_toast-container__8fZtQ stack_gap-xs__xTnny"
       id="toast-container"
       data-testid="toast-container"
       style="padding: 0px;"
  >
  </div>
</main>
`;
